torch/_prims/__init__.py

```diff
@@ -2731,6 +2731,8 @@ def _svd_meta(
     is_cuda = A.device.type == "cuda"
     strides_Vh = utils.make_contiguous_strides_for(shape_Vh, row_major=is_cuda)
     Vh = TensorMeta(shape=shape_Vh, strides=strides_Vh, dtype=A.dtype, device=A.device)
+    if A.numel() != 0 and is_cuda and Vh.is_complex():
+        Vh = Vh.conj()
```

Comment:
Not sure this belongs to a meta func, but this fixes issues with the `conj` bit not being set in some cases.

In core, this is not part of the meta func. However, when we do these checks in tests, we also execute the core impl. So this fix is based on:

- a `if (A.numel() == 0)` short-circuit in `TORCH_IMPL_FUNC(_linalg_svd_out)` in core
- the following from `svd_kernel` in `aten/src/ATen/native/cuda/linalg/BatchLinearAlgebra.cpp`:

```cpp
    if (compute_uv && Vh.is_complex()) {
      Vh._set_conj(!Vh.is_conj());
    }
```

Reply:
Hmm, we have not thought carefully about the interaction of conj bit and meta functions. Since we do model conj bit on meta tensors, this seems fine to do. Weird that this is CUDA only though.

Reply:
The CPU and CUDA backends of SVD return different things, and we have to normalize them ourselves. Some return V while others return V^H. Even different cuSolver backends return different things. This is why we need all this conj and stride dance.